frontend/common/stores/account-store.js

```diff
@@ -381,8 +385,12 @@ const store = Object.assign({}, BaseStore, {
       return false
     }
 
+    if (store.samlOrOauth) {
```

Comment:
Can be replaced with my other suggestion.

Reply:
This prevents the 2factor from being forced on Oauth or Saml accounts